Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/ApplicationDataStorageHelper.cs

```diff
@@ -0,0 +1,324 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Helpers;
+using Windows.Storage;
+using Windows.System;
+
```

Comment:
The UWP target doesn't support the nullable switch from the .csproj, so unfortunately ne need to manually enable it with a precompiler directive in each source file containing nullable annotations. We'll be able to remove this workaround when we switch to just supporting WinUI 3 (as that's on .NET 5), but we need this for now 😄

```suggestion

#nullable enable

```

Reply:
Ok, I was wondering about that. I dabbled with adding `<Nullable>enable</Nullable>` to the csproj, but that didn't seem like the right thing to do.